apps/docs/static/data/releases.json

```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "title": "New environment flag improves deploy cache use",
+    "product": "WebOps",
+    "fileName": "2022-09-10-webops",
+    "description": "WebOps can now reuse pull requests' cache for new commit deploys. This is because of a new environment flag you can set in the `vtex.env` file of your FastStore project: `USE_BUILD_CACHE`."
```

Comment:
```suggestion
    "description": "WebOps can now reuse cache from previous commits in newer deploy previews. This is because of a new environment flag you can set in the `vtex.env` file of your FastStore project: `USE_BUILD_CACHE`."
```